### DISCUSSION TITLE
broken link on repository for 'documentation'

### DISCUSSION THREAD

**Discussion by cboyda** [community/community#55779] [Repositories]
### Select Topic Area

Bug

### Body

I am looking at my repository.
URL = https://github.com/cboyda/LighthouseLabs/tree/main/Project-SQL
Lower left is github links for DOCUMENTATION and SHARE FEEDBACK
Clicking 'documentation' loads https://docs.github.com/en/repositories/working-with-files/managing-files/navigating-files-with-the-new-code-view
aka
Ooops!
It looks like this page doesn't exist.

We track these errors automatically, but if the problem persists please feel free to contact us.

**Comment by anxumalo** [comment]
Good spot @cboyda .. let's hope it gets fixed soon! 👍🏼 

At first I was sure the left-hand panel was from an older preview to display the **File tree** but then I realised that was in the Pull Request "Files Changed" panel - not the same as what you described. I eventually figured out I needed to enable the feature preview for **Global navigation update** to see the real feature - "Code View". 🎉

**Reply by cboyda** [reply]
lol ya I should have included a screenshot, surprised the error 404 page doesn't auto-do that... instead the 'contact support' link just goes into an automated frustration cycle :)
